recipe/meta.yaml

```diff
@@ -6,63 +6,65 @@ package:
   version: {{ version }}
 
 source:
-  fn: opencv-{{ version }}.tar.gz
-  url: https://github.com/opencv/opencv/archive/{{ version }}.tar.gz
-  sha256: f1b87684d75496a1054405ae3ee0b6573acaf3dad39eaf4f1d66fdd7e03dc852
+  - url: https://github.com/opencv/opencv/archive/{{ version }}.tar.gz
+    fn: opencv-{{ version }}.tar.gz
+    sha256: f1b87684d75496a1054405ae3ee0b6573acaf3dad39eaf4f1d66fdd7e03dc852
+    patches:  # [win]
+      - 01_win_fix_cmake_config_sufix.patch  # [win]
+  - url: https://github.com/opencv/opencv_contrib/archive/{{ version }}.tar.gz
+    fn: opencv_contrib-{{ version }}.tar.gz
+    sha256: 298c69ee006d7675e1ff9d371ba8b0d9e7e88374bb7ba0f9d0789851d352ec6e
+    folder: opencv_contrib-{{ version }}
 
 build:
-  number: 200
+  number: 204
   # Python2.7 support dropped: https://github.com/opencv/opencv/issues/8481
   skip: true  # [win and py27]
-  features:                    # [not win]
+  features:  # [not win]
     - blas_{{ blas_variant }}  # [not win]
 
 requirements:
   build:
     - python
     - toolchain
-    - pkgconfig                    # [unix]
-    # For sha256 comparisons of opencv_contrib
-    - openssl 1.0.*                # [unix]
-    # For downloading opencv_contrib
-    - curl >=7.44.0,<8
     # For applying patches
-    - git                          # [win]
+    - m2-patch  # [win]
+    - pkgconfig  # [unix]
     - cmake
-    - numpy 1.8.*   # [not (win and (py35 or py36))]
-    - numpy 1.9.*   # [win and py35]
+    - numpy 1.8.*  # [not (win and (py35 or py36))]
+    - numpy 1.9.*  # [win and py35]
     - numpy 1.11.*  # [win and py36]
```

Comment:
`numpy` can be simplified to use our modern pinning strategy.

ref: https://conda-forge.org/docs/meta.html#building-against-numpy